ansible/create/roles/create-cfme-appliance/files/create-cfme-appliance.py

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env python
```

Comment:
I am of the opinion we should house the script in a different directory.  Typically the files directory in ansible means a file you'll transfer.  In this case, we are using a "helper" script.  I think we should house this in: ansible/scripts directory (This directory doesn't exist yet.)  We should just double check and make sure it can run fine froma lower directory as such.